test/cluster.sh

```diff
@@ -55,18 +55,26 @@ function install_knative_serving() {
     build_knative_from_source
     INSTALL_ISTIO_CRD_YAML="${ISTIO_CRD_YAML}"
     INSTALL_ISTIO_YAML="${ISTIO_YAML}"
-    # TODO(#2122): Use RELEASE_YAML once we have monitoring e2e.
-    INSTALL_RELEASE_YAML="${RELEASE_NO_MON_YAML}"
+    # TODO(#2122): Install monitoring as well once we have e2e testing for it.
+    INSTALL_RELEASE_YAML="${SERVING_YAML}"
   fi
+  # TODO: Should we install build from a release?
+  INSTALL_BUILD_YAML=third_party/config/build/release.yaml
```

Comment:
This might break at least the upgrade tests. You're hardcoding the build manifest to the one in the source, but upgrade tests install serving from a published release (currently v0.2.3). Probably the best thing to do here is to pass the build manifest as the 4th parameter (and update any calls to this function).

Reply:
It's strange to be using the vendored build manifest here at all, rather than the released version of Build (which should have its own upgrade tests).

I've switched this to use the latest released build for now... I could switch this to use a build nightly if you'd prefer.